src/extensions/DC/getDCEntity.js

```
@@ -130,6 +130,18 @@ const getKind = type => {
 	return 'postType';
 };
 
+const getUsersByType = async (type, args) => {
+	const users = await resolveSelect('core').getUsers(args);
+
+	if (type === 'customers') {
+		return users.filter(
+			user => user.roles && user.roles.includes('customer')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)